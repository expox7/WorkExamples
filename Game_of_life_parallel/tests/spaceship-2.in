7  9  30  2  2  ! height  width  max_gen  n_rows n_cols
.........
..X.X....
.....X...
.....X...
..X..X...
...XXX...
.........
