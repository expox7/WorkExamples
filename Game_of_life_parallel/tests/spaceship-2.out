Generation 1
.........
..X.X....
.....X...
.....X...
..X..X...
...XXX...
.........

Generation 2
.........
.........
....XX...
....XXX..
...X.XX..
...XXX...
....X....

Generation 3
.........
.........
....X.X..
...X.....
...X.....
...X..X..
...XXX...

Generation 4
....X....
.........
.........
...XX....
..XXX....
..XX.X...
...XXX...

Generation 5
...XXX...
.........
.........
..X.X....
.....X...
.....X...
..X..X...

Generation 6
...XXX...
....X....
.........
.........
....XX...
....XXX..
...X.XX..

Generation 7
...X..X..
...XXX...
.........
.........
....X.X..
...X.....
...X.....

Generation 8
..XX.X...
...XXX...
....X....
.........
.........
...XX....
..XXX....

Generation 9
.....X...
..X..X...
...XXX...
.........
.........
..X.X....
.....X...

Generation 10
....XXX..
...X.XX..
...XXX...
....X....
.........
.........
....XX...

Generation 11
...X.....
...X.....
...X..X..
...XXX...
.........
.........
....X.X..

Generation 12
...XX....
..XXX....
..XX.X...
...XXX...
....X....
.........
.........

Generation 13
..X.X....
.....X...
.....X...
..X..X...
...XXX...
.........
.........

Generation 14
.........
....XX...
....XXX..
...X.XX..
...XXX...
....X....
.........

Generation 15
.........
....X.X..
...X.....
...X.....
...X..X..
...XXX...
.........

Generation 16
.........
.........
...XX....
..XXX....
..XX.X...
...XXX...
....X....

Generation 17
.........
.........
..X.X....
.....X...
.....X...
..X..X...
...XXX...

Generation 18
....X....
.........
.........
....XX...
....XXX..
...X.XX..
...XXX...

Generation 19
...XXX...
.........
.........
....X.X..
...X.....
...X.....
...X..X..

Generation 20
...XXX...
....X....
.........
.........
...XX....
..XXX....
..XX.X...

Generation 21
..X..X...
...XXX...
.........
.........
..X.X....
.....X...
.....X...

Generation 22
...X.XX..
...XXX...
....X....
.........
.........
....XX...
....XXX..

Generation 23
...X.....
...X..X..
...XXX...
.........
.........
....X.X..
...X.....

Generation 24
..XXX....
..XX.X...
...XXX...
....X....
.........
.........
...XX....

Generation 25
.....X...
.....X...
..X..X...
...XXX...
.........
.........
..X.X....

Generation 26
....XX...
....XXX..
...X.XX..
...XXX...
....X....
.........
.........

Generation 27
....X.X..
...X.....
...X.....
...X..X..
...XXX...
.........
.........

Generation 28
.........
...XX....
..XXX....
..XX.X...
...XXX...
....X....
.........

Generation 29
.........
..X.X....
.....X...
.....X...
..X..X...
...XXX...
.........

Generation 30
.........
.........
....XX...
....XXX..
...X.XX..
...XXX...
....X....

